document_test.go

```diff
@@ -287,12 +286,10 @@ func TestDocument_GetCharRelativeToCursor(t *testing.T) {
 		},
 	}
 
-	for i, p := range patterns {
+	for _, p := range patterns {
 		ac := p.document.GetCharRelativeToCursor(1)
 		ex, _ := utf8.DecodeRuneInString(p.expected)
-		if ac != ex {
-			t.Errorf("[%d] Should be %s, got %s", i, string(ex), string(ac))
-		}
+		assert.Equal(t, string(ex), string(ac))
```

Comment:
Let's update the old tests in a separate commit. This change does not belong to the current commit.

Reply:
Added separate commit.